test/e2e/app-dir/dynamic/app/dynamic/named-export/page.js

```diff
@@ -0,0 +1,11 @@
+import dynamic from 'next/dynamic'
+
+const Button = dynamic(() =>
```

Comment:
https://github.com/vercel/next.js/issues/49454#issuecomment-1830053413

What does `dynamic()` do between Server Component and Client Component?